src/index.html

```diff
@@ -10,8 +10,203 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+    class="form"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()"
+    autocomplete="off"
+    >
```

Comment:
Для атрибутов нужны отступы
```suggestion
    <form
      class="form"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      onsubmit="onSubmit()"
      autocomplete="off"
    >
```